README.md

```diff
@@ -11,8 +11,69 @@ it to 'deploy' directory of target nexus repository manager. For more informatio
 "[Installing a custom Nexus 3 plugin](https://sonatype-nexus-community.github.io/nexus-development-guides/plugin-install.html)".
 
 # Usage
-This plugin provides tagging functionality by exposing RESTFul API. See swagger documentation in System->API section of 
-Administration panel to see how this can be used.
+This plugin provides tagging functionality by exposing RESTFul API. Each tag is a document with a unique name and a
```

Comment:
Is it necessary to tell that each tag is a document? i think it's confusing for user of this plugin who has not any knowledge of the database and ... .

Each tag has a unique name and a set of attributes. it can be associated with one or more available components in sonatype nexus. Tags can be searched by their names, attributes or assiciated components.

Reply:
Done